sdk/confidentialledger/Azure.Security.CodeTransparency/src/CodeTransparencyClient.cs

```diff
@@ -101,68 +106,122 @@ bool CertValidationCheck(X509Certificate2 cert)
             return new HttpPipelineTransportOptions { ServerCertificateCustomValidationCallback = args => CertValidationCheck(args.Certificate) };
         }
 
-        /// <summary> Post an entry to be registered on the CodeTransparency instance. The returned operation will have the entry id when it is complete. </summary>
-        /// <param name="body"> A raw CoseSign1 signature. </param>
-        /// <param name="cancellationToken"> The cancellation token to use. </param>
-        /// <exception cref="ArgumentNullException"> <paramref name="body"/> is null. </exception>
-        public virtual Operation<GetOperationResult> CreateEntry(BinaryData body, CancellationToken cancellationToken = default)
```

Comment:
It was implemented in such a way was to be able to do polling on the operation as the generated code could not do that for our custom implementation, i.e. `return new CreateEntryOperation(this, result.OperationId)`

Reply:
Reimplemented the CreateEntry APIs manually, with polling included